Firestore/Protos/cpp/google/protobuf/empty.pb.h

```diff
@@ -0,0 +1,206 @@
+/*
```

Comment:
This includes definitions of common protos that are included in libprotobuf (but not libprotobuf-lite). We needed to delete these from the Objective-C built--should we do the same here?